tests/unit/blocking_connection_tests.py

```diff
@@ -5,6 +5,10 @@
 """
 import socket
 
+import errno
```

Comment:
It's easier on the eyes (for some developers, at least :) ) if the imports within each group are sorted, so in this case, you might have:

```
import errno
import socket
```